logs/parse.go

```diff
@@ -541,13 +511,69 @@ func ParseLogLineWithPrefix(prefix string, line string) (logLine state.LogLine,
 	return
 }
 
+func getOccurredAt(timePart, timeFormat, timeFormatAlt string, tz *time.Location) time.Time {
+	if tz != nil {
+		result, err := time.ParseInLocation(timeFormat, timePart, tz)
+		if err == nil {
+			return result
+		}
+		if timeFormatAlt != "" {
+			// Ensure we have the correct format remembered for ParseInLocation call that may happen later
+			timeFormat = timeFormatAlt
```

Comment:
If we fail here and go to the old logic, `timeFormat` is updated as `timeFormatAlt` on line 531 `ts, err := time.Parse(timeFormat, timePart)` part already. I think we could pass `timeFormatAlt` below instead.

Reply:
Good catch. This should be taken care of once I address the changes mentioned above.